package.json

```diff
@@ -22,58 +23,21 @@
   "devDependencies": {
     "@stoplight/types": "1.0.x",
     "@types/chance": "1.0.x",
-    "@types/express": "4.16.x",
     "@types/jest": "23.3.x",
-    "@types/node": "10.11.x",
+    "@types/node": "10.12.x",
     "ajv": "6.5.x",
     "chance": "1.0.x",
     "jest": "23.6.x",
+    "oclif": "^1.12.5",
```

Comment:
Shouldn't it be runtime dependency of cli package?

Reply:
Oh, no, that's just a CLI for `building` the CLI :)

It's like the `@angular/cli`

Reply:
:+1: